.rhcicd/floorist/floorplan.yaml

```diff
@@ -216,11 +216,68 @@ objects:
           INNER JOIN
             bundles
               ON bundles.id = applications.bundle_id
+- apiVersion: metrics.console.redhat.com/v1alpha1
+  kind: FloorPlan
+  metadata:
+    name: notifications-hms
+  spec:
+    database:
+      secretName: ${FLOORIST_DB_SECRET_NAME}
+    objectStore:
+      secretName: ${FLOORIST_HMS_BUCKET_SECRET_NAME}
+    logLevel: ${FLOORIST_LOGLEVEL}
+    suspend: ${{FLOORIST_SUSPEND}}
+    disabled: ${{FLOORIST_DISABLED}}
+    queries:
+    - prefix: hms_analytics/notifications/${FLOORIST_ENV_PREFIX}/notifications_deliveries
+      chunksize: 1000
+      query: >-
+        SELECT
+          bun.display_name::TEXT AS bundle,
+          apps.display_name::TEXT AS application,
+          e.org_id::TEXT,
+          CASE
+            WHEN
+              nh.endpoint_type_v2 = 'CAMEL'
+            THEN
+              LOWER(nh.endpoint_sub_type)
+            ELSE
+              LOWER(nh.endpoint_type_v2)
+          END AS endpoint_type,
+          et.name::TEXT AS event_type,
+          nh.status::TEXT,
+          e.created::DATE,
+          COUNT(*) AS count
```

Comment:
Just a comment just in case! Here you would be getting the events from our internal customers too, since the aggregation occurs in the database itself. I guess that this is fine with your use case, but just wanted to mention it in case you want to filter out the internal customers from the external ones.

This is why we had to have our queries not aggregated...

Reply:
It should aggregate it by org_id as well. I understand the reason for NOT aggregating, and I'd personally prefer it. The issue is that it would create lots of unnecessary data for us.

Reply:
Understood! Thanks for the explanation!